<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordToolTip</class>
 <widget class="EntityWidget" name="PasswordToolTip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>111</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>271</width>
    <height>111</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>271</width>
    <height>111</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
    background-color: transparent;
}</string>
  </property>
  <property name="respondable">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>263</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QFrame {
    border: 1px solid #00aeef;
    background-color: #ffffff;
}

QLabel {
font: 11px &quot;Open Sans&quot;;
color: black;
}
QProgressBar {
    border: none;
	background-color: #bbbbbb;
margin: 0px 0px 0px 0px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>7</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item alignment="Qt::AlignTop">
     <widget class="QWidget" name="title" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="msg_password_strenght">
         <property name="styleSheet">
          <string notr="true">QLabel {
font: 550 12px &quot;Open Sans&quot;;
}
</string>
         </property>
         <property name="text">
          <string>@msg_pass_strenght:</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="InfoMessage" name="msg_reason">
         <property name="styleSheet">
          <string notr="true">font: 12px &quot;Open Sans&quot;;</string>
         </property>
         <property name="text">
          <string notr="true">@msg_to_short</string>
         </property>
         <property name="messages">
          <stringlist>
           <string>@msg_too_short</string>
           <string>@msg_passrestore</string>
           <string>@msg_weakpass</string>
           <string>@msg_midpass</string>
           <string>@msg_goodpass</string>
           <string>@msg_strongpass</string>
          </stringlist>
         </property>
         <property name="styles">
          <stringlist notr="true">
           <string>QLabel { color: #191919; }</string>
           <string>QLabel { color: #373737; }</string>
           <string>QLabel { color: #371919; }</string>
           <string>QLabel { color: #373719; }</string>
           <string>QLabel { color: #193719; }</string>
           <string>QLabel { color: #195119; }</string>
          </stringlist>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QProgressBar" name="progressBar">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>6</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>6</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk {
    background-color: #de123a;
}</string>
      </property>
      <property name="value">
       <number>26</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="format">
       <string/>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QLabel" name="desc_password_strenght">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>@text_password_description</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="triangle">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>36</y>
     <width>12</width>
     <height>38</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources.qrc">:/textures/passtip_triangle.png</pixmap>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EntityWidget</class>
   <extends>QWidget</extends>
   <header>plugins.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InfoMessage</class>
   <extends>QLabel</extends>
   <header>plugins.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
